pkg/partition/partition.go

```diff
@@ -40,6 +40,9 @@ const (
 
 	// NoOfLogicalBlocksForGPTHeader is the no. of logical blocks for the GPT header.
 	NoOfLogicalBlocksForGPTHeader = 1
+	
+	// openEbsNDMPartitionName is the name meta info for openEBS created partitions.
+	const openEbsNDMPartitionName = "OpenEBS_NDM"
```

Comment:
```suggestion
	openEbsNDMPartitionName = "OpenEBS_NDM"
```

Reply:
One more suggestion @avats-dev

Reply:
We dont need to use the keyword const again, as it is already in the `const` block